YogaAdventures.js

```diff
@@ -0,0 +1,312 @@
+    }
+}
+
+/**
+ * GAME OVER/RESTART
+ * Invites the user to play again.
+ * Calls process.exit() to end the game.
+ */
+const gameOver = () => {
+    console.log("")
+    if(rls.keyInYNStrict("Would you like to play another round?")) {
+        gameLoop()
+    }
+    console.log("See you soon then!")
+    process.exit()
+}
+
+/**
+ * QUIT GAME
+ * Calls process.exit() to quit the game.
+ */
+const quitGame = () => {
+    console.log("")
+    console.log("Good luck dealing with stress! GOOD BYE!")
+    process.exit()
+ }
+
+/**
+ * WELCOME/START/QUIT
+ * Welcomes the user.
+ * Starts the game by prompting the user to play.
+ * Calls either startGame() or quitGame().
+ */
+const name = rls.question("What's your name?"); {
+    console.log("")
+    console.log("Hello, " + name + "!" + " Welcome to Yoga Adventures!")
+    console.log("")
+    if(rls.keyInYNStrict("Are you a current Pursuit Fellow?")) {
+        console.log("")
+        console.log("Excellent! Yoga Adventures is a great tool to help maintain equanamity as you journey through the program.")
+        startGame()
+    } else {
+        console.log("Congratulations on completing the program!")
+        console.log("")
+        console.log("Yoga Adventures is a great tool to help current Fellows maintain equanamity as they journey through the program.")
+        console.log("")
+        console.log("You're more than welcome to use this tool also.")
+        if(rls.keyInYNStrict("Would you like to continue?")) {
+            startGame()
+        } else {
+            quitGame()
+        }
+    }
+}
+
+/**
+ * HEEL POSE
+ */
+const heelPose = () => {
+    console.log("")
+    console.log("Start on your mat by sitting on your heels. Relax your body from your head to your toes.")
+    console.log("Rest your palms either FACE DOWN or FACE UP on your thighs.")
+    console.log("")
+    console.log("We'll do this three times: Take a deep breath in through your nose and open mouth exhale. \nRelease any thoughts that won't serve you on this journey of success.")
+    console.log("")
+    console.log("Inhale love, gratitude, and an 'I CAN' energy. \nNOTE: You can take this 'I CAN' energy with you everywhere you go.")
+    console.log("")
+    console.log("Take a few more deep breaths in and out.")
+    console.log("")
+    if(rls.keyInYNStrict("Do you need more time?")) {
+        console.log("")
+        console.log("Take all the time you need.")
+        console.log("")
+        rls.keyIn("Press the SPACEBAR when you're done.")
+    } else {
+        console.log("Wonderful!")
+    }
+}
+
+/**
+ * PALM UP/DOWN
+ */
+const palmQuestion = () => {
+    let palmUpDownResp = ""
+    do {
+        palmUpDownResp = rls.question("During HEEL POSE, were your palms FACE UP (UP) or FACE DOWN (DOWN)?")
+    }
+    while (palmUpDownResp !== "UP" && palmUpDownResp !== "DOWN") {
```

Comment:
Cool you used a do while loop. 
Helpful tip, update your users input. I typed in "up" but it didn't work. 
Us .toLowerCase() or .toUpperCase and .trim() to help improve your users experience